app/microbreak.js

```diff
@@ -36,8 +36,7 @@ window.onload = (e) => {
     const progressTime = document.querySelector('#progress-time')
     const postponeElement = document.querySelector('#postpone')
     const closeElement = document.querySelector('#close')
-    const mainColor = settings.get('mainColor')
```

Comment:
I see this is causing issue (try for example White theme), As I described here, this is not the same value: https://github.com/hovancik/stretchly/issues/1084#issuecomment-1126977211